addons/wowl/static/src/components/dropdown/dropdown.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+  <div class="o_dropdown" t-name="wowl.Dropdown" t-on-mouseenter.prevent="onMouseEnter" t-on-mouseleave.prevent="onMouseLeave" t-on-dropdown-item-clicked="onItemClicked" owl="1">
```

Comment:
line too long (soft limit: 80, hard limit: 100)
should be caught by prettier (?)

Reply:
I did distribute the attrs over multiple lines, but I confirm that prettier puts it back on a oneliner state...